CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ flowchart LR
 If you want to preview the whole documentation site on your machine:
 
 ```sh
-cd mermaid
+cd packages/mermaid
```

Comment:
This is how I got it to work at least, but would be great if somebody else could confirm this.